common/facility.go

```diff
@@ -0,0 +1,53 @@
+package common
+
+// Facility maps facility numeric codes to facility string messages.
+var Facility = map[uint8]string{
+	0:  "kernel messages",
+	1:  "user-level messages",
+	2:  "mail system",
+	3:  "system daemons",
+	4:  "security/authorization messages",
+	5:  "messages generated internally by syslogd",
+	6:  "line printer subsystem",
+	7:  "network news subsystem",
+	8:  "UUCP subsystem",
+	9:  "clock daemon",
+	10: "security/authorization messages",
+	11: "FTP daemon",
+	12: "NTP subsystem",
+	13: "log audit",
+	14: "log alert",
+	15: "clock daemon (note 2)", // (todo) > some sources reporting "scheduling daemon"
```

Comment:
I want to remove it too and I'll remove it.

Anyway I left `note 2` here in the past because there are two keys (9, 15) pointing to 2 clock daemons, as you also noted below.

Reply:
Done